spec/features/person_audit_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'View person audit' do
+  let(:super_admin_email)  { 'test.user@digital.justice.gov.uk' }
+  let!(:super_admin)  { create(:super_admin, email: super_admin_email) }
+
+  let(:description)  { 'The best person' }
+  let(:phone_number) { '55555555555' }
+  let!(:person)      { with_versioning { create(:person) } }
+
+  before do
+    with_versioning do
+      person.update description: description
+      person.update primary_phone_number: phone_number
+    end
+  end
+
+  scenario 'View audit as an admin user' do
+    omni_auth_log_in_as(super_admin.email)
+    visit person_path(person)
+
+    expect(page).to have_selector 'table.audit tbody tr', count: 3
```

Comment:
I'm not gonna push for it, but I introduced `SitePrism` to the project some time ago - to abstract html specifics from the feature specs. If you're interested and think it's a good idea, I'd move this to the page object. 
It may end up being a bit broader discussion about using page objects, of course.